src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -1225,7 +1233,7 @@
     <value>Rename</value>
     <comment>Text label for a button that can be used to begin the renaming process.</comment>
   </data>
-  <data name="ColorScheme_DeleteButtonDisclaimerInBox.Text" xml:space="preserve">

```

Comment:
We should **prefer** this kind of resource: a UID with a `.Property`. XAML automatically wires them up with the right `x:Uid`.

Loading these xaml UI string resources via code should be a last resort!